data/agent-groups.yaml

```diff
@@ -22,4 +22,16 @@
   name: Agent Group SIP
   settings:
     TServer:
-      gvm_mailbox: '8888'
\ No newline at end of file
+      gvm_mailbox: '8888'
+- managerDBIDs:
+    - 104
+  agentDBIDs:
+    - 104
+  DBID: 107
+  tenantDBID: 1
+  name: Agent Group Test
+  settings:
+    TServer:
+      gvm_mailbox: '8888'
+    studio:
+      modules: xtend-workspace
```

Comment:
This is added to show that the Studio can merge configuration.

Reply:
I dont't think we should push the configuration options for the moment as the feature isn't official.